apiserver/params/network.go

```diff
@@ -577,11 +582,12 @@ type AddSubnetsParams struct {
 // SubnetProviderId must be set, but not both. Zones can be empty if
 // they can be discovered
 type AddSubnetParams struct {
-	SubnetTag        string   `json:"subnet-tag,omitempty"`
-	SubnetProviderId string   `json:"subnet-provider-id,omitempty"`
-	SpaceTag         string   `json:"space-tag"`
-	VLANTag          int      `json:"vlan-tag,omitempty"`
-	Zones            []string `json:"zones,omitempty"`
+	SubnetTag         string   `json:"subnet-tag,omitempty"`
+	SubnetProviderId  string   `json:"subnet-provider-id,omitempty"`
```

Comment:
(as discussed on IRC) 
Unfortunately these names are already inconsistent in the codebase. I picked this version based on the existing ones in `github.com/juju/juju/network/network.go`: 
On `InterfaceInfo`
* ProviderSubnetId
* ProviderSpaceId
* ProviderVLANId
* ProviderAddressId

On `SubnetInfo`:
* SpaceProviderId

I could name it NetworkProviderId on `network.SubnetInfo` and `params.AddSubnetArgs` (and ProviderNetworkId elsewhere) so it's consistent within a type, but that seems more confusing (and means more tech debt work if/when we decide to make them all consistent).